Database-Files/dummy.sql

```diff
@@ -1,393 +1,550 @@
+    values ('HCM', 3);
 
-insert into training(name, description, trainingType)
-values ('People Manager Toolkit', 'Learn how to manage little baby developers', 'Technical skills');
-insert into training(name, description, trainingType)
-    values ('SA Development', 'Learn the responsibilities required in your new role', 'Professional skills');
-insert into training(name, description, trainingType)
-values ('Kainos as a Business', 'Learn how Kainos business processes operate', 'Development programmes');
-insert into training(name, description, trainingType)
-values ('Influencing Others', 'Learn how to work with a team and positively influence your colleagues', 'Professional skills');
-insert into training(name, description, trainingType)
-values ('Coaching - Foundation', 'Learn the basics of mentoring and teaching colleagues', 'Technical skills');
-insert into training(name, description, trainingType)
-values ('Consultant Development', 'Learn the responsibilities required in your new role', 'Professional skills');
-insert into training(name, description, trainingType)
-values ('Certified Scrum Master', 'Learn how to lead an agile team', 'Development programmes');
-
-
-insert into competency(title)
-values ('Commercial Awareness');
-insert into competency(title)
-values ('Communicating & Teamwork');
-insert into competency(title)
-values ('Innovation & Continous Improvement');
-insert into competency(title)
-values ('Customer Focus');
-insert into competency(title)
-values ('Developing Yourself & Others');
-insert into competency(title)
-values ('Planning & Organising');
-insert into competency(title)
-values ('Job Specific Knowledge');
-
-
-insert into responsibilities (respDesc)
-values ('Develop talent within your team, ensuring SMARTER stretch and project goals are clearly defined');
-insert into responsibilities (respDesc)
-values ('Provide anytime performance feedback on Workday to each of your team members at least once a quarter');
-insert into responsibilities (respDesc)
-values ('Identify training needs for team members and report to the BU talent manager');
-insert into responsibilities (respDesc)
-values ('Participate, and encourage your people to participate, in recruitment activities');
-insert into responsibilities (respDesc)
-values ('Deliver training as part of MAP');
-insert into responsibilities (respDesc)
-values ('Encourage collaboration across BUs');
-insert into responsibilities (respDesc)
-values ('Identify and mentor high-potential, junior talent and expedite their personal development');
-insert into responsibilities (respDesc)
-values ('Represent Kainos at careers fairs or Kainos open evenings events if invited');
-insert into responsibilities (respDesc)
-values ('Deliver your tasks within the timelines while adhering to the Kainos quality standards');
-insert into responsibilities (respDesc)
-values ('Immediately tell your manager if your tasks are not going to be complete within the expected timeframe');
```

Comment:
Where are we entering these now

Reply:
Same as above, the responsibilities/comp are entered for each band as it is created etc